canonical-web-and-design/maas-ui
# feat(ui): add bridge form

## Done

- Add a form to create a bridge to the network tab.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the react network tab for a machine.
- Tick an interface and click "Create bridge".
- Fill out the form and submit.
- The bridge should get created.

## Fixes

Fixes: #2153.
